draft-ietf-lake-edhoc-psk.md

```diff
@@ -417,22 +418,37 @@ Before sending message_3 the Initiator can derive PRK_out and create an OSCORE-p
 
 # Security Considerations
 
-The EDHOC-PSK authentication method introduces deviations from the initial specification of EDHOC {{RFC9528}}. This section analyzes the security implications of these changes.
+The EDHOC-PSK authentication method introduces deviations from the initial specification of EDHOC {{RFC9528}}. This section analyzes the security implications of these changes and specifies the security properties of EDHOC authenticated with PSK, following those of {{I-D.ietf-lake-reqs}}.
 
 ## Identity Protection
 
 In EDHOC-PSK, ID_CRED_PSK in message_3 is encrypted with a keystream derived from the ephemeral shared secret G_XY. As a result, unlike the asymmetric authentication methods in {{Section 9.1 of RFC9528}}, which protect the Initiator’s identity against active attackers and the Responder’s identity against passive attackers, EDHOC-PSK protects both the Initiator and the Responder identities against passive attackers.
 
 ## Mutual Authentication
 
+At the end of the protocol, each endpoint SHALL have freshly authenticated the other's credential if a fourth message is sent, even with long-term key compromise (Key Compromise Impersonation), resisting
+active Man-in-the-Middle (MitM) adversaries, replay attacks and identity misbinding. In particular, both endpoints MUST agree on a fresh session identifier, and the roles and credentials of both endpoints.
+
```

Comment:
>even with long-term key compromise (Key Compromise Impersonation),
With compromise of the long-term key, the attacker can trivially impersonate both parties. PSK does not provide KCI